### PR TITLE
K8s filestorage bastion

### PR DESCRIPTION
**Pull request by d3vil-st**

### CODE REVIEW

File: kubernetes/terraform/kubernetes-filestore/bastion.tf
Comment (malibora) [inline kubernetes/terraform/kubernetes-filestore/bastion.tf:1]: Please make this a standalone module we can reuse in any other k8s terraform solution. As an example you can have a look at loki module I am working on at the moment: https://github.com/nebius/nebius-architect-solution-library/tree/feature/log-aggregation/kubernetes/terraform/loki

Then in this k8s-fileshare, we can introduce it in main.tf, something like:

module "bastion" {
  providers = {
    nebius = nebius
  }
  count = var.bastion? 1:0
  folder_id = var.folder_id
  ...

}

**Review by malibora — CHANGES_REQUESTED** [review]

File: kubernetes/terraform/kubernetes-filestore/bastion.tf
Comment (d3vil-st) [reply kubernetes/terraform/kubernetes-filestore/bastion.tf:1]: Done.

File: kubernetes/terraform/kubernetes-filestore/main.tf
Comment (malibora) [inline kubernetes/terraform/kubernetes-filestore/main.tf:1]: Pls uncomment kubernetes-training, and comment kubernetes-inference

File: kubernetes/terraform/kubernetes-filestore/terraform.tfvars
Comment (malibora) [inline kubernetes/terraform/kubernetes-filestore/terraform.tfvars:1]: pls set explicitly node_count to 2... clients wont be aware of this if not visible in tfvars...

**Review by malibora — CHANGES_REQUESTED** [review]

**Review by malibora — APPROVED** [review]

**Pull request merged**